t/mldistwatch-misc.t

```diff
@@ -252,6 +252,39 @@ subtest "check various forms of version" => sub {
   );
 };
 
+subtest "check overlong versions" => sub {
+  my $pause = PAUSE::TestPAUSE->init_new;
+  $pause->import_author_root('corpus/mld/long-version/authors');
+  my $result = $pause->test_reindex;
+
+  $pause->file_not_updated_ok(
+    $result->tmpdir
+           ->file(qw(cpan modules 02packages.details.txt.gz)),
+    "there were no things to update",
+  );
+
+  my $etoolong = sub {
+    like(
+      $_[0]{email}->get_body,
+      qr/Version string is exceeds maximum allowed length/,
```

Comment:
The typo I mentioned above is also present in the test here. `string is exceeds` should be `string exceeds`